cmd/autoowners/main.go

```diff
@@ -404,3 +409,28 @@ func main() {
 		logrus.WithError(err).Fatal("PR creation failed.")
 	}
 }
+
+type ownersCleaner func([]string) []string
+
+func ownersCleanerFactory(ghc github.OrganizationClient) (ownersCleaner, error) {
+	members, err := ghc.ListOrgMembers(githubOrg, "all")
+	if err != nil {
+		return nil, fmt.Errorf("listOrgMembers failed: %w", err)
+	}
+
+	membersSet := sets.String{}
+	for _, member := range members {
+		membersSet.Insert(member.Login)
+	}
+
+	return func(unfilteredMembers []string) []string {
```

Comment:
nit:

```go
return func(unfilteredMembers []string) []string {
    return sets.NewString(unfilteredMembers...).Difference(membersSet).List()
}
```

Reply:
Intersection, not Difference, that returns the items that are in membersSet and not in unfilteredMembers